integration-tests/mtls-certificates/src/main/resources/c-role-mappings.txt

```diff
@@ -0,0 +1,2 @@
+AU=user,admin
```

Comment:
IMHO, it is quite unrealistic, which indirectly implies that capturing so many various attributes in the enum may be unnecessary

Reply:
I am fine with a String as long as for SAN it is a string for a known values (AKA enum :-)). Please suggest config property names because apart of fixing ASN1 parsing, I am ready to submit this PR for review. I am open to suggestion, just need something more concrete.

Reply:
btw I agree that RNDs like `STREET` is quite unrealistic :-)

Reply:
I dropped this test as you suggested it's unlikely scenario. I'll close this comment thread then.